.github/actions/spelling/expect.txt

```diff
@@ -20,6 +20,8 @@ benbi
 bigfish
 bigquery
 blockdevices
+blocklist
+blocklisting
```

Comment:
These two should go in our main dictionary file https://github.com/mondoohq/spellcheck-dictionary